documentation/src/main/asciidoc/topics/con_xsite_auto_offline.adoc

```diff
@@ -1,9 +1,15 @@
 [id='cross-site-automatic-offline_{context}']
-= Factors that take backup locations offline automatically
+= Automatic offline parameters for backup locations
+
+When network issues occur or remote sites become unavailable, {brandname} can automatically take those backup locations offline.
+This prevents {brandname} from continually re-trying failed cross-site replication operations, resulting in excessive CPU usage for nodes.
+
+You can configure {brandname} to automatically take backup locations offline using:
+
+* Timeout values
```

Comment:
"Timeout values" seems too abstract to me, both here and as a header. And I'm not sure the xsite timeout belongs here, because
1. The timeout is also used to report failures to the application, with sync xsite
2. The request can also fail because the remote site doesn't have any node in the bridge cluster, long before the timeout expired (right @pruivo ?)

IMO the intro should say something about why this "take offline" stuff is needed at al, e.g.

{brandname} does not detect when a remote site is unavailable directly: the failure of individual nodes is not relevant, as other nodes from the remote site may later replace the failed node in the bridge cluster.

Still, sending and re-sending backup requests is expensive, so {brandname} can take a remote site offline when it stops accepting backup requests for a long time. (NB. tracking the incomplete requests also costs RAM, not just CPU)

{brandname} decides when to take a remote site offline based on the number of failed sequential requests and the time interval since the first failure. A request is considered failed when the target site does not have any nodes in the bridge cluster, or when a timeout expires before the target site acknowledges the request.

Reply:
yes, Infinispan fails immediately if not relay node from the remote site is live.

Reply:
@danberindei Great comment. I've made some changes. I've changed the second level heading to "Backup timeouts" to match the Infinispan configuration and help with scan-ability.